nl_server/requirements.txt

```diff
@@ -0,0 +1,3 @@
+Flask==1.1.4
```

Comment:
We will certainly need this (I can also add them later):

torch==1.12.1
pandas==1.3.5
spacy==3.4.4
en_core_web_md @ https://github.com/explosion/spacy-models/releases/download/en_core_web_md-3.4.1/en_core_web_md-3.4.1-py3-none-any.whl

Reply:
ok, sg, we can add those as you migrate them